src/eventSchemas/explorer/schemaItemNode.ts

```diff
@@ -26,6 +28,13 @@ export class SchemaItemNode extends AWSTreeNodeBase {
             dark: Uri.file(globals.iconPaths.dark.schema),
             light: Uri.file(globals.iconPaths.light.schema),
         }
+        this.command = !isCloud9()
+            ? {
+                  command: 'aws.viewSchemaItem',
+                  title: localize('AWS.command.viewSchemaItem', 'Open Schema'),
+                  arguments: [this],
+              }
+            : undefined
```

Comment:
Shouldn't need to gate this on C9 as the service itself is gated. 
```suggestion
        this.command = {
              command: 'aws.viewSchemaItem',
              title: localize('AWS.command.viewSchemaItem', 'Open Schema'),
              arguments: [this],
          }
```

Reply:
Not really. We mostly figure out what works (and what doesn't) by just running the extension on Cloud9. Sometimes the feature partially works but the UX isn't great so it's gated anyway. Some of the feature gates probably aren't needed anymore too